packages/host/app/app.ts

```diff
@@ -10,6 +10,11 @@ import '@cardstack/boxel-ui/styles/global.css';
 import '@cardstack/boxel-ui/styles/fonts.css';
 import '@cardstack/boxel-ui/styles/variables.css';
 import 'ember-power-select/styles';
+import { registerDateLibrary } from 'ember-power-calendar';
+import DateUtils from 'ember-power-calendar-moment';
+import 'ember-power-calendar/styles';
+
+registerDateLibrary(DateUtils);
```

Comment:
do you know if this registration mechanism is fastboot friendly? (otherwise our indexer will choke on it)